src/ResourceManagement/Graph.RBAC/Generated/Models/Page.cs

```diff
@@ -24,7 +24,7 @@ public class Page<T> : IPage<T>
          /// <summary>
          /// Gets the link to the next page.
          /// </summary>
-         [JsonProperty("nextLink")]
+         [JsonProperty("odata.nextLink")]
```

Comment:
Seems graphrbac is not generated for long (or the swagger get fixed in meantime).